packages/wrangler/src/config/environment.ts

```diff
@@ -479,6 +479,12 @@ interface EnvironmentNonInheritable {
 			type: string;
 			[key: string]: unknown;
 		}[];
+
+		metadata:
```

Comment:
This will need a JSDoc comment to describe it, similar to the one above `bindings`.

Reply:
Done.

Reply:
Sorry, I meant in front of the `metadata` property 😄.